db-shared/src/main/java/com/splicemachine/db/shared/common/reference/SQLState.java

```diff
@@ -1986,6 +1986,7 @@ The message identifier (ie. the value that matches a field in this class)
 	String TABLE_NOT_PINNED				    						= "EXT32";
 	String DIRECTORY_REQUIRED			    						= "EXT33";
 	String ACCESSING_S3_EXCEPTION                                   = "EXT34";
+	String COLS_PERMS_EXCEPTION                                     = "EXT35";
 
```

Comment:
EXT* is for external table related exception. You may want to put it close to authorization/authentication excpetions